src/ja/server/main.py

```diff
@@ -30,6 +33,7 @@ def _init_algorithm() -> SchedulingAlgorithm:
     @staticmethod
     def _read_config() -> ServerConfig:
         with open('/etc/jobadder/server.conf') as f:
+            logger.info("reading server.conf config file")
```

Comment:
reading `/etc/jobadder/server.conf` config file ... You could have a thousand different server.confs on your computer, but only one of them is loaded!